wls-vorfaelleundvorkommnisse-service/src/test/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisControllerIntegrationTest.java

```diff
@@ -0,0 +1,170 @@
+import de.muenchen.oss.wahllokalsystem.wls.common.security.Profiles;
+import de.muenchen.oss.wahllokalsystem.wls.common.testing.SecurityUtils;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.val;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.transaction.annotation.Transactional;
+
+import static de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.TestConstants.SPRING_TEST_PROFILE;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(classes = MicroServiceApplication.class, webEnvironment = SpringBootTest.WebEnvironment.MOCK)
+@AutoConfigureMockMvc
+@ActiveProfiles(profiles = { SPRING_TEST_PROFILE, Profiles.NO_BEZIRKS_ID_CHECK })
+public class EreignisControllerIntegrationTest {
+
+    @Autowired
+    MockMvc api;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    @Autowired
+    EreignisRepository ereignisRepository;
+
+    @Autowired
+    EreignisDTOMapper ereignisDTOMapper;
+
+    @Autowired
+    EreignisModelMapper ereignisModelMapper;
+
+    @AfterEach
+    void tearDown() {
+        SecurityUtils.runWith(Authorities.REPOSITORY_DELETE_EREIGNISSE);
+        ereignisRepository.deleteAll();
+    }
+
+    @Nested
+    class GetEreignisse {
+
+        @Test
+        @WithMockUser(authorities = { Authorities.SERVICE_GET_EREIGNISSE, Authorities.REPOSITORY_READ_EREIGNISSE })
+        @Transactional
+        void should_return_empty_response_when_no_data_found() throws Exception {
+            val request = MockMvcRequestBuilders.get("/businessActions/ereignisse/wahlbezirkID");
+            val response = api.perform(request).andExpect(status().isNoContent()).andReturn();
+
+            Assertions.assertThat(response.getResponse().getContentAsString()).isEmpty();
+        }
+
+        @Test
+        @WithMockUser(authorities = { Authorities.SERVICE_GET_EREIGNISSE, Authorities.REPOSITORY_READ_EREIGNISSE, Authorities.REPOSITORY_WRITE_EREIGNISSE })
+        @Transactional
+        void should_return_WahlbezirkEreignisseDTO_when_data_found() throws Exception {
```

Comment:
passt aus meiner Sicht besser zu deinen asserts

```suggestion
        void should_return_persisted_WahlbezirkEreignisseDTO_when_data_found() throws Exception {
```

Reply:
siehe commit [50cad96](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/50cad968e13e70c97c398f5338f1d8d08882e7df)